README.md

```diff
@@ -114,8 +144,14 @@ Instructions for developers for testing [here](docs/TESTING.md). If you want to
 + Telegram: https://t.me/joinmarketorg
 
 ### Support JoinMarket and bitcoin privacy
+---
 
 Donate to help make JoinMarket even better: [Obtain a bitcoin address here](https://bitcoinprivacy.me/joinmarket-donations)
 
 JoinMarket is an open source project which does not have a funding model, fortunately the project itself has very low running costs as it is almost-fully decentralized and available to everyone for free. Developers contribute only as volunteers and donations are divided amongst them. Many developers have also been important in advocating for privacy and educating the wider bitcoin user base. Be part of the effort to improve bitcoin privacy and fungibility. Every donated coin helps us spend more time on JoinMarket instead of doing other stuff.
 
+### Other useful links
+---
+1. [Visit In-detail documentation on Github pages](https://joinmarket-org.github.io/joinmarket-clientserver/)
```

Comment:
I would make these two links more prominent, so probably near or at the top of the ToC under some title like 'Documentation'.

It's a bit confusing, but we've for years had our main documentation stored under the `/docs` subdirectory on this github repo, in the form of markdown files. This is fine, and people can use it, but it is a bit preferable, especially for the more non-developer users, to have them stored on standard html pages, so I'm keen to promote the fact that the documentation exists on that static github pages site.

Reply:
Updated.